src/univalue/include/univalue.h

```diff
@@ -137,6 +139,14 @@ class UniValue {
     friend const UniValue& find_value( const UniValue& obj, const std::string& name);
 };
 
+template <class It>
+bool UniValue::push_backV(It first, It last)
+{
+    if (typ != VARR) return false;
```

Comment:
Probably should throw here instead of return a value that is likely to be unchecked (as below)?